chef_master/source/chef_license_accept.rst

```diff
@@ -0,0 +1,196 @@
+=====================================================
+Accepting the Chef License
+=====================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/chef_license_embedded.rst>`__
+
+This page aims to document how to accept the Chef license for all Chef Software products. For an overview of the license
+see the `Chef license </chef_license.html>`__ documentation.
+
+Workstation Products
+=====================================================
+* Chef Workstation, which also contains
+  * Chef Infra
+  * InSpec
+  * Push Jobs Client
+* Chef Infra
+* InSpec
```

Comment:
Do we want to use our new marketing name here?

Reply:
doh, good catch, update incoming